examples/common/builders.py

```diff
@@ -67,9 +68,9 @@ def build_optimizer(cfg, model):
                               weight_decay=cfg.weight_decay)
     elif cfg.name == 'decoupled_lionw':
         return DecoupledLionW(model.parameters(),
-                    lr=cfg.lr,
-                    betas=cfg.betas,
-                    weight_decay=cfg.weight_decay)
+                              lr=cfg.lr,
+                              betas=cfg.betas,
+                              weight_decay=cfg.weight_decay)
```

Comment:
kind of a conflict with https://github.com/mosaicml/examples/pull/206#discussion_r1122528816